### ISSUE TITLE
Reset opcache via CLI

### ISSUE THREAD

**Issue by Seldaek** [EasyCorp/easy-deploy-bundle#1]
https://github.com/gordalina/cachetool/ actually lets you reset opcaches from CLI at least if you use FCGI/FPM. It does this by creating a temp file and then instructing the FCGI socket to execute that file, so it all happens without public URL. It would be good to somehow integrate it as an option, maybe if the binary is available globally it could be used, or require it as a dependency?

**Comment by javiereguiluz** [comment]
Thanks for the proposal. I saw the tool when looking for solutions to clear the OPcache from the CLI. Sadly, as you said, it's only for people using FPM. But yes, I'm going to look how we can integrate that. Thanks!

**Comment by Seldaek** [comment]
Yeah it probably can't be the only solution you support if you care about apache, but it's a good way to simplify things for FPM users.
